<ui version="4.0" >
 <class>SnapGuideConfigWidget</class>
 <widget class="QWidget" name="SnapGuideConfigWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>59</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QCheckBox" name="useSnapGuides" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="snapDistance" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Proximity distance where snapping happens</string>
       </property>
       <property name="suffix" >
        <string> px</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>4</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" >
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QCheckBox" name="nodeSnapGuide" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Nodes</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset>
         <normaloff>pics/ox22-action-hand.png</normaloff>pics/ox22-action-hand.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QCheckBox" name="extensionSnapGuide" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Extension of lines</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset>
         <normaloff>pics/ox22-action-path-break-point.png</normaloff>pics/ox22-action-path-break-point.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QCheckBox" name="boundingBoxSnapGuide" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Bounding box of shape</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset>
         <normaloff>pics/ox22-action-path-break-segment.png</normaloff>pics/ox22-action-path-break-segment.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QCheckBox" name="orthogonalSnapGuide" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Orthogonal</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset>
         <normaloff>pics/ox22-action-hand.png</normaloff>pics/ox22-action-hand.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QCheckBox" name="lineGuideSnapGuide" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Guide lines</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset>
         <normaloff>pics/ox22-action-path-break-segment.png</normaloff>pics/ox22-action-path-break-segment.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QCheckBox" name="intersectionSnapGuide" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Intersection of lines</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset>
         <normaloff>pics/ox22-action-path-break-point.png</normaloff>pics/ox22-action-path-break-point.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
